### PR TITLE
Update Player.cs

### PR DESCRIPTION
**Pull request by manamini**
just a test

### CODE REVIEW

File: Keeper/Keeper/VisualControlers/Player/Player.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 namespace Keeper.VisualControlers.Player {
     class Player : IVisualControl {
-
+                                            //this is just a test to make sure I am not an idiot
```
Comment (NicholasBusby) [inline Keeper/Keeper/VisualControlers/Player/Player.cs:11]: one nice comment you have there.

**Comment by manamini** [comment]
i thought it was good

File: Keeper/Keeper/VisualControlers/Player/Player.cs (hunk 1 above)
Comment (NicholasBusby) [inline Keeper/Keeper/VisualControlers/Player/Player.cs:11]: see how the line note keeps things together.

Comment (manamini) [inline Keeper/Keeper/VisualControlers/Player/Player.cs:11]: hey would ya look at that... a line note

**Pull request merged**